### PR TITLE
add houdini.md zh-cn version

### PR DESCRIPTION
**Pull request by liuliangsir**
I add houdini.md zh-cn version,thanks

### CODE REVIEW

**Comment by liuliangsir** [comment]
I signed it! @googlebot

File: src/content/en/updates/posts/2016/05/2016-06-05-Houdini-Demystifying-Css.md [hunk 1]
```diff
@@ -0,0 +1,186 @@
+你是否曾思考过(在页面布局中)CSS所承载的工作量？(紧接着)你修改了某个CSS属性的属性值，然后你整个网站瞬间以不同的layout展示在我们面前。CSS在页面布局方面确实有点魔力。(谁能告诉我，拥有这些魔力我们今后的道路将如何走?)到目前为止，身在web开发者社区的我们才能见证以及体验这些魔力。如果我们想提出自己的魔力该怎么办？如果我们想变成魔法师该怎么办？加入Houdini工作小组！
```
Comment (PaulKinlan) [inline src/content/en/updates/posts/2016/05/2016-06-05-Houdini-Demystifying-Css.md]: It looks like we are missing all of the required meta-data for this update.

Also, it is in the wrong folder, it should be in the zh folder if I remeber correctly.

cc @surma

File: src/content/en/updates/posts/2016/05/houdini-zh-cn-version.md [hunk 2]
```diff
@@ -0,0 +1,202 @@
+---
+layout: updates/post
```
Comment (petele) [inline src/content/en/updates/posts/2016/05/houdini-zh-cn-version.md]: Should not be included in translated versions.

File: src/content/en/updates/posts/2016/05/houdini-zh-cn-version.md [hunk 3]
```diff
@@ -0,0 +1,202 @@
...
+title: "Houdini – Demystifying CSS"
+description: "Houdini is a collection of APIs that expose the CSS engine’s internals to developers"
+published_on: 2016-05-20
```
Comment (petele) [inline src/content/en/updates/posts/2016/05/houdini-zh-cn-version.md]: Should not be included in translated versions.

File: src/content/en/updates/posts/2016/05/houdini-zh-cn-version.md [hunk 4]
```diff
@@ -0,0 +1,202 @@
...
+updated_on: 2016-05-20
+authors:
```
Comment (petele) [inline src/content/en/updates/posts/2016/05/houdini-zh-cn-version.md]: Should not be included in translated versions.

File: src/content/en/updates/posts/2016/05/houdini-zh-cn-version.md [hunk 5]
```diff
@@ -0,0 +1,202 @@
...
+featured_image: /web/updates/images/2016/05/houdini/compworklet_small.png
+
+---
+
+
+你是否曾思考过(在页面布局中)CSS所承载的工作量？(紧接着)你修改了某个CSS属性的属性值，然后你整个网站瞬间以不同的layout展示在我们面前。CSS在页面布局方面确实有点魔力。(谁能告诉我，拥有这些魔力我们今后的道路将如何走?)到目前为止，身在web开发者社区的我们才能见证以及体验这些魔力。如果我们想提出自己的魔力该怎么办？如果我们想变成魔法师该怎么办？加入Houdini工作小组！  
+
+Houdini工作小组聚齐了来自Mozilla，Apple，Opera，Microsoft，HP，Intel以及Google工程师,工作小组的意图旨在把CSS引擎的部分模块暴露给web开发者。Houdini工作小组正在制订能够被W3C组织认可从而逐渐成为web标准的草案集。Houdini工作小组给他们自己定下了更高的目标，要将(由他们制订的)草案集变成规范草案，反过来，(由他们制订的)草案将会变成低层次且带有辅助性质的草案集。当有人谈论到"Houdini"时通常就表示他们谈论的是草案集。在我们写(自己筹备的)草案集的那段时间，草案列表都是处于未完成状态，甚至(草案列表中的)一些草案仅仅还是处于计划阶段。可见我们Houdini工作小组的准备工作是有多么的早啊。  
+
+![Houdini Compositor Worklet for transitions](http://7xl614.com1.z0.glb.clouddn.com/20160607095542.png)
```
Comment (petele) [inline src/content/en/updates/posts/2016/05/houdini-zh-cn-version.md]: Please use the original link to the video, not a screenshot that isn't linkable.

File: src/content/en/updates/posts/2016/05/houdini-zh-cn-version.md [hunk 6]
```diff
@@ -0,0 +1,202 @@
...
+
+
+你是否曾思考过(在页面布局中)CSS所承载的工作量？(紧接着)你修改了某个CSS属性的属性值，然后你整个网站瞬间以不同的layout展示在我们面前。CSS在页面布局方面确实有点魔力。(谁能告诉我，拥有这些魔力我们今后的道路将如何走?)到目前为止，身在web开发者社区的我们才能见证以及体验这些魔力。如果我们想提出自己的魔力该怎么办？如果我们想变成魔法师该怎么办？加入Houdini工作小组！  
+
+Houdini工作小组聚齐了来自Mozilla，Apple，Opera，Microsoft，HP，Intel以及Google工程师,工作小组的意图旨在把CSS引擎的部分模块暴露给web开发者。Houdini工作小组正在制订能够被W3C组织认可从而逐渐成为web标准的草案集。Houdini工作小组给他们自己定下了更高的目标，要将(由他们制订的)草案集变成规范草案，反过来，(由他们制订的)草案将会变成低层次且带有辅助性质的草案集。当有人谈论到"Houdini"时通常就表示他们谈论的是草案集。在我们写(自己筹备的)草案集的那段时间，草案列表都是处于未完成状态，甚至(草案列表中的)一些草案仅仅还是处于计划阶段。可见我们Houdini工作小组的准备工作是有多么的早啊。  
+
+![Houdini Compositor Worklet for transitions](http://7xl614.com1.z0.glb.clouddn.com/20160607095542.png)  
+
+免责声明：我想先放出Houdini草案的预览版，所以你可能会想Houdini工作小组究竟想解决哪些问题。只要目前制定的规范能够被浏览器部分支持，我会试着给出代码实例。请记住这些规范还是只是草案，而且很不稳定。所以不能保证这些代码案例在不久的将来能够完全正确，否则只有等这些草案变成现实中的标准,这样才能保证代码案例完全正确。
+##规范###
```
Comment (petele) [inline src/content/en/updates/posts/2016/05/houdini-zh-cn-version.md]: Markdown tags should have a space after them, and do not include hashtags at the end. For example: ## 规范

File: src/content/en/updates/posts/2016/05/houdini-zh-cn-version.md [hunk 7]
```diff
@@ -0,0 +1,202 @@
...
+###Worklets([规范](http://https://drafts.css-houdini.org/worklets/))###
```
Comment (petele) [inline src/content/en/updates/posts/2016/05/houdini-zh-cn-version.md]: Markdown tags should have a space after them, and do not include hashtags at the end.

Comment (petele) [inline src/content/en/updates/posts/2016/05/houdini-zh-cn-version.md]: Also double http/https here. If an HTTPS version is available, always use that over HTTP.

File: src/content/en/updates/posts/2016/05/houdini-zh-cn-version.md [hunk 8]
```diff
@@ -0,0 +1,202 @@
...
+Worklets对自己本身来说用处不大。因为Worklets只是尽可能的为以后的草案提供内容介绍的支持。在你阅读"worklet"的同时想到Web Workers，就会发现你(前面的论述)是正确的。因为他们之间有太多的概念交叉。因此我们可能会有这个疑问，在我们已经有web workers的前提下为什么还要有Worklet？Houdini旨在暴露新的API给web开发者，让他们可以将自己的代码挂载在CSS引擎或者外围环境。假定fragments中的一些片段必须运行所有单帧的想法是不现实的。有些代码片段是需要被定义。引用[Web Worker规范：](https://www.w3.org/TR/workers/)
+>Workers[...]，相对来说比较笨重，以及在处理大数方面不推荐使用。例如针对含有四百万像素的图片中的每个像素都开启一个线程的类似做法是不可取的。  
+
+这就意味着web workers跟Houdini要做的事情不匹配。因此worklets就应运而生。Worklets使用ES2015的class来定义一个包含方法的集合，通过使用worklet的数据类型提前定义方法的签名。而且Worklets具有轻量以及生存周期短的特点。  
+##Paint Worklet([规范](https://drafts.css-houdini.org/css-paint-api/))  
+正是因为Paint Worklets介绍的新概念少，所以我才开始研究Paint Worklets。下面内容是从Paint Worklets的规范草案摘抄出来的  
+>CSS的paint脚手架主要是负责背景的绘制以及在元素的几何结构(用来产生layout脚手架)和样式基础上来绘制内容和实现高亮。
```
Comment (petele) [inline src/content/en/updates/posts/2016/05/houdini-zh-cn-version.md]: Need spaces after markdown tags

**Comment by petele** [comment]
Hi,
Thanks for your contribution! I've added a few comments that need to be addressed before we can merge this. In addition, this file needs to be moved to `src/content/zh-cn/updates/posts/2016/05/houdini.md` so it can be properly served as a localized file.  

Please take a look at our contributions.md and translations.md docs for more info!

Thanks!
PEte

**Comment by petele** [comment]
Closing this as it's been quiet for a while. Please feel free to re-open once the issues are addressed.

**Pull request closed without merging**